csm/cli/schema/health.json

```diff
@@ -0,0 +1,87 @@
+{
+    "name": "health",
+    "description": "Health",
+    "sub_commands": [
+      {
+        "name": "show",
+        "description": "Displays health of the cluster and resources in it.",
+        "need_confirmation": false,
+        "permissions_tag": "list",
+        "args": [
+          {
+            "flag": "resource",
+            "type": "str",
+            "help": "The type of element in the hierarchy. Ex. node"
```

Comment:
Is there a list of choices for `resource` ? What are the possible values for these argumant?

Reply:
@udayan-y21 
yes, there is a list of choices. for eg. [cluster, site, rack , node] and in future that list will grow. 
And as discussed with @pawankumarsrivastava, possible list of arguments will be mentioned in CLI command specification documents.